README.md

```diff
@@ -317,6 +328,63 @@ tendrl-ansible:
     on [TEN-257](https://tendrl.atlassian.net/browse/TEN-257)).
 
 
+## How do I cleanup Tendrl setup using tendrl-ansible?
+
+1)  If you use tendrl-ansible from rpm package, copy `cleanup.yml` playbook into
```

Comment:
Main information we are missing here is what does this cleanup mean and why would one want to do one. This helps to convey context of this feature.

Reply:
Also it seems that the cleanup deletes all tendrl internal data (etcd, graphite, ...). It seems to me that this needs to be highlighted here again.

Reply:
Ack